cfpb/cfgov-refresh
# Allow users to set the width of snippet columns

Snippet lists that live aside each other often have inconsistent column
widths. Allowing the user to set the width of the Assets column makes
the layouts more consistent.

## Additions

- Snippet width column setting
- Test for width
- Atomic helpers for test

## Testing

1. Run `tox -e fast v1.tests.atomic_elements.organisms.test_organisms`
2. Create a browse page locally
3. Add a snippet list and set the column width to a two digit number (ex `40`)
4. Navigate to the page and ensure the column is honoring your width.

## Screenshots

![screen shot 2017-10-12 at 9 18 54 am](https://user-images.githubusercontent.com/1280430/31506829-38594ca0-af3e-11e7-84e2-1174262d5ce6.png)

## Checklist
